### PR TITLE
Consistent content for essential requirements

### PR DESCRIPTION
**Pull request by idavidmcdonald**
A content tweak for the following pivotal story - https://www.pivotaltracker.com/story/show/129839965

Content changes outlined in [this google doc](https://docs.google.com/document/d/1TDZhOjV9ykfPA0tMoWbxS8oZzyGXG8B7lOF51SRRaO4/edit?ts=5852b310)

Tests will fail until the related [frameworks pull request](https://github.com/alphagov/digitalmarketplace-frameworks/pull/346) is merged.

For the start page we were using the word "criteria". We now use "requirements"
image

For the essential requirements met page we use "essential skills and experience" instead of "essential requirements.
image

### CODE REVIEW

File: app/templates/briefs/start_brief_response.html [hunk 1]
```diff
@@ -48,9 +48,9 @@ <h3 class="heading-small">Evidence structure</h3>
           <li>what the results were</li>
         </ul>
       </div>
-      <p>There’s a 100-word maximum for each essential or nice-to-have criteria.</p>
-      <p>You should only provide one example for each essential or nice-to-have criteria (unless the buyer specifies otherwise).</p>
-      <p>You can reuse examples across different essential or nice-to-have criteria if you need to.</p>
+      <p>There’s a 100-word maximum for each essential or nice-to-have requirement.</p>
+      <p>You should only provide one example for each essential or nice-to-have requirement (unless the buyer specifies otherwise).</p>
+      <p>You can reuse examples across different essential or nice-to-have requirement if you need to.</p>
```
Comment (Wynndow) [inline app/templates/briefs/start_brief_response.html]: should be `requirements`

Comment (idavidmcdonald) [inline app/templates/briefs/start_brief_response.html]: eagled eyed, nice!

**Comment by idavidmcdonald** [comment]
Have fixed the missing "s"

**Comment by Wynndow** [comment]
👍

**Pull request merged**